openwisp_radius/tests.py

```diff
@@ -169,6 +169,30 @@ def test_register_404(self):
         })
         self.assertEqual(r.status_code, 404)
 
+    def test_api_show_only_token_org(self):
+        org = Organization.objects.create(name='org1')
+        rad = OrganizationRadiusSettings.objects.create(token='asdfghjklqwertg',
+                                                        organization=org)
+        self.assertEqual(self.radius_accounting_model.objects.count(), 0)
+        nas_ip = '127.0.0.1'
+        test1 = self.radius_accounting_model(session_id='asd1',
+                                             organization=org,
+                                             nas_ip_address=nas_ip,
+                                             unique_id='123')
+        test1.full_clean()
+        test1.save()
+        test2 = self.radius_accounting_model(session_id='asd2',
+                                             organization=self.default_org,
+                                             nas_ip_address=nas_ip,
+                                             unique_id='1234')
+        test2.full_clean()
+        test2.save()
+        data = self.client.get(self._acct_url,
+                               HTTP_AUTHORIZATION='Bearer {0} {1}'.format(org.pk,
```

Comment:
shouldn't this authorization header already be present in the test class since it's used in all other tests?